spec/presenters/coronavirus/sub_section_json_presenter_spec.rb

```diff
@@ -62,15 +67,16 @@
         expect(sub_sections_list).to include(
           hash_including(
             url: path,
+            label: content,
+            description: summary,
             featured_link: true,
-            description: description,
           ),
         )
       end
 
-      it "sets a nil description for an absolute featured link" do
+      it "sets a nil description for an absolute action link url" do
```

Comment:
I don't think this test is correct either as it's up to the user to set the description an an action link url.

Reply:
This test has been removed.